resources/qml/TimelineRow.qml

```diff
@@ -292,4 +295,17 @@ AbstractButton {
         reactions: r.reactions
         eventId: r.eventId
     }
+
+    Rectangle {
+        id: unreadRow
+        anchors {
+            top: reactionRow.bottom
+            topMargin: 5
+        }
+        color: Nheko.colors.highlight
+        width: row.maxWidth
+        visible: (r.isRoomUnread && (r.fullyReadEventId == r.eventId))
```

Comment:
Why do we need this condition? Couldn't we just make this depend on if this is index 0 or not? I.e. don't render it if the fully read marker is the last event?